### DISCUSSION TITLE
How to look for larger features across images?

### DISCUSSION THREAD

**Discussion by neriniccolodei** [OpenStitching/stitching#171] [Q&A]
Hello, 

Is there a way to tell sift or orb to look for "large" features among two images? I have tried to lower the scaling factor (medium size), but that doesn't really help. I have these images obtained from almost pure camera rotation. Stitching should be straightforward, yet I cannot really obtain anything good. 

Moreover, I was curious to ask if the pipeline automatically looks for features on black and white versions of the images or if they're processed as RGB?

Best regards

![frame000001](https://github.com/OpenStitching/stitching/assets/62433425/4d615a50-a38e-4e1e-9806-33965d9ffb88)
![frame000002](https://github.com/OpenStitching/stitching/assets/62433425/393c967d-65a1-4d49-af32-295ffae2ddf2)

**Comment by lukasalexanderweber** [comment]
I think you should consider pre processing to obtain your cameras.

I've set the contrast to min and saturation to max, and the algorithm has a better time finding features

[1](https://github.com/OpenStitching/stitching/assets/32765578/a7bc3bbb-dac7-4792-b97a-2e0affd8a6d2)
[2](https://github.com/OpenStitching/stitching/assets/32765578/078a0775-69ba-4934-b936-ca21de7d01d4)

`stitch * -v --medium_megapix 0.1`

![01_features_img1](https://github.com/OpenStitching/stitching/assets/32765578/fa90f9ec-21f9-4dea-80d9-1baf822242b8)

vs 

![01_features_img1](https://github.com/OpenStitching/stitching/assets/32765578/99490b6f-3417-42c0-926b-e24f0d8bfb3c)



> Moreover, I was curious to ask if the pipeline automatically looks for features on black and white versions of the images or if they're processed as RGB?

as RGB

**Reply by neriniccolodei** [reply]
Thank you. What did you use to change contrast and brightness?

**Reply by lukasalexanderweber** [reply]
In this case just Windows Photo Viewer :D

**Reply by neriniccolodei** [reply]
Thank you, works much better.
